tests/Unit/SubscriptionEventTest.php

```diff
@@ -0,0 +1,193 @@
+<?php
+namespace ChartMogul\Tests;
+
+use ChartMogul\Http\Client;
+use ChartMogul\SubscriptionEvent;
+use GuzzleHttp\Psr7;
+use GuzzleHttp\Psr7\Response;
+
+class SubscriptionEventTest extends TestCase
+{
+    const ALL_SUBSCRIPTION_EVENT_JSON = '{
+    "subscription_events": [
+      {
+        "id": 73966836,
+      	"data_source_uuid": "ds_1fm3eaac-62d0-31ec-clf4-4bf0mbe81aba",
+      	"customer_external_id": "scus_023",
+      	"subscription_set_external_id": "sub_set_ex_id_1",
+      	"subscription_external_id": "sub_0023",
+      	"plan_external_id": "p_ex_id_1",
+      	"event_date": "2022-04-09T11:17:14Z",
+      	"effective_date": "2022-04-09T10:04:13Z",
+      	"event_type": "subscription_cancelled",
+      	"external_id": "ex_id_1",
+      	"errors": {},
+      	"created_at": "2022-04-09T11:17:14Z",
+      	"updated_at": "2022-04-09T11:17:14Z",
+      	"quantity": 1,
+      	"currency": "USD",
+      	"amount_in_cents": 1000
+      },
+      {
+        "id": 73966837,
+      	"data_source_uuid": "ds_1fm3eaac-62d0-31ec-clf4-4bf0mbe81aba",
+      	"customer_external_id": "scus_024",
+      	"subscription_set_external_id": "sub_set_ex_id_2",
+      	"subscription_external_id": "sub_0024",
+      	"plan_external_id": "p_ex_id_2",
+      	"event_date": "2022-04-09T11:17:14Z",
+      	"effective_date": "2022-04-09T10:04:13Z",
+      	"event_type": "subscription_cancelled",
+      	"external_id": "ex_id_2",
+      	"errors": {},
+      	"created_at": "2022-04-09T11:17:14Z",
+      	"updated_at": "2022-04-09T11:17:14Z",
+      	"quantity": 1,
+      	"currency": "USD",
+      	"amount_in_cents": 1000
+      }
+    ],
+    "current_page": 1,
+    "total_pages": 1
```

Comment:
1. The pagination attributes should reside under the `meta` key
2. The pagination on this endpoint differs from others, for example there is no `current_page` attribute